MarkdownView.js

```diff
@@ -45,7 +45,7 @@ function mergeRules(baseRules, rules) {
   return mergedRules
 }
 
-const IMAGE_LINK = "(?:\\[[^\\]]*\\]|[^\\]]|\\](?=[^\\[]*\\]))*"
+const IMAGE_LINK = "(?:\\[[^\\]]*\\]|[^\\[\\]]|\\](?=[^\\[]*\\]))*";
```

Comment:
This fixes a potential regex DoS: See https://github.com/Khan/simple-markdown/issues/29